tests/e2e/cli/test_cli.py

```diff
@@ -133,7 +133,6 @@ def test_otx_e2e_cli(
     assert (latest_dir / "csv").exists()
 
     # 3) otx export
-    fxt_export_list = []
     if task in ("visual_prompting", "zero_shot_visual_prompting"):
         fxt_export_list.append(ExportCase2Test("ONNX", False, "exported_model_decoder.onnx"))
         fxt_export_list.append(ExportCase2Test("OPENVINO", False, "exported_model_decoder.xml"))
```

Comment:
```suggestion
        fxt_export_list = [
            ExportCase2Test("ONNX", False, "exported_model_decoder.onnx"),
            ExportCase2Test("OPENVINO", False, "exported_model_decoder.xml"),
        ]
```
Coluld you udpate anomaly part below like this?

Reply:
I understood `fxt_export_list` to be a fixture that takes an argument. So isn't the current change right?

Reply:
`fxt_export_list` is a list including which cases to test for export. As default, `fxt_export_list` includes ONNX, OV and exportable code. But vp, zvp and anomaly tasks don't support exportable_code so it needs to be excluded. If new items are appended into `fxt_export_list` as current implementation then fxt_export_list  includes 5 cases and these 5 cases will be tested, which isn't intended, I think.